variables.tf

```diff
@@ -525,7 +525,7 @@ variable "create_fargate_logging_policy_for_karpenter" {
 variable "karpenter_chart_version" {
   description = "Chart version for Karpenter"
   type        = string
-  default     = "v0.34.6"
+  default     = "0.37.1"
```

Comment:
```suggestion
  default     = "v0.37.1"
```

Reply:
As we discussed, we cannot pass as v because public ecr is without v